pkgs/by-name/wf/wfview/package.nix

```diff
@@ -0,0 +1,65 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitLab,
+  kdePackages,
+  eigen,
+  hidapi,
+  libopus,
+  libpulseaudio,
+  portaudio,
+  rtaudio,
+}:
+
+stdenv.mkDerivation (finalAttr: {
+  pname = "wfview";
+  version = "2.10";
+
+  src = fetchFromGitLab {
+    owner = "eliggett";
+    repo = "wfview";
+    rev = "v${finalAttr.version}";
+    hash = "sha256-bFTblsDtFAakbSJfSfKgvoxd1DTSv++rxU6R3/uWo+4=";
+  };
+
+  patches = [
+    # Remove syscalls during build to make it reproducible
+    # We also need to adjust some header paths for darwin
+    ./remove-hard-encodings.patch
+  ];
+
+  buildInputs = (
+    [
+      eigen
+      hidapi
+      libopus
+      libpulseaudio
+      portaudio
+      rtaudio
+    ]
+    ++ (with kdePackages; [
+      qtbase
+      qcustomplot
+      qtmultimedia
+      qtserialport
+      qtwebsockets
+    ])
+  );
+
+  nativeBuildInputs = with kdePackages; [
+    wrapQtAppsHook
+    qmake
+  ];
+
```

Comment:
On Darwin `wfview` is installed in `$out/bin/wfview.app/Contents/MacOS/wfview`, which is not optimal. It's better to do something like this in `postInstall`:

https://github.com/NixOS/nixpkgs/blob/0922a79cf39fcb7944ae1aaf2099ff6a4f904500/pkgs/applications/networking/instant-messengers/discord/darwin.nix#L58-L70

In short, we'd move the bundle to `$out/Applications/wfview.app` and make a wrapper of `$out/Applications/wfview.app/Contents/MacOS/wfview` at `$out/bin/wfview`.